markdown.dtx

```diff
@@ -23699,12 +23703,25 @@ function M.reader.new(writer, options)
                      * Cb("level")
                      / writer.heading
 
-  parsers.SetextHeading = #(parsers.line * S("=-"))
-                        * Ct(parsers.linechar^1
-                            / self.parser_functions.parse_inlines)
-                        * parsers.newline
+  function remove_enclosing_whitespaces(o)
```

Comment:
This should trigger a Luacheck error (W111: setting non-standard global variable 'remove_enclosing_whitespaces'). It's unclear to me why GitHub Actions is not running checks over your PRs.

```suggestion
  local function remove_enclosing_whitespaces(o)
```